doc/classes/EditorPlugin.xml

```diff
@@ -214,6 +210,12 @@
 			<argument index="0" name="event" type="InputEvent">
 			</argument>
 			<description>
+				This method is called when there is an input event in the 2D viewport, e.g. the user clicks with the mouse in the 2D space (canvas item GUI). Keep in mind that for this method to be called you have to first declare the virtual method [method handles] so the editor knows that you want to work with the workspace:
+				[codeblock]
+				func handles(object):
+				    return true
+				[/codeblock]
+				Also note that the edited scene must have a root node.
```

Comment:
`Also, the edited scene must have a root node for this to work.`

Is it specific to this method or in general, for the class to work? If it's a general requirement for editor plugins, this should be in the description.

Reply:
No, this is only necessary for this method. Some plugins work with files in the FileSystem docs.